src/components/mapView/mapView.module.scss

```diff
@@ -0,0 +1,17 @@
+@use "breakpoints";
+
+// Px amount is roughly adjusted by hand
+.mapView {
+  grid-column: 1 / -1 !important;
+  height: calc(100vh - 120px);
+  width: 100%;
+  z-index: 1;
+
+  @include breakpoints.respond-above(s) {
+    height: calc(100vh - 130px);
+  }
+
+  @include breakpoints.respond-above(m) {
+    height: calc(100vh - 140px);
+  }
```

Comment:
They tweak the MapView in a way that the the map container doesn't overflow the screen. Mainly because the map contains links to openstreet etc, so those are visible all the time.

Reply:
Would it be useful to document this in the file? It would also be interesting to understand where the values come from--why they need to be different for different breakpoints etc.

Reply:
I'll add some comments to the file, but for example breakpoint-s makes the main header slimmer as the logo height gets smaller. Without this the footers container would be shown a little above the buttons and the black border wouldn't cover the whole height.